gk/main.c

```diff
@@ -256,6 +256,19 @@ pkt_copy_cached_eth_header(struct rte_mbuf *pkt, struct ether_cache *eth_cache,
 	return stale;
 }
 
+static inline void
+prefetch0_128_bytes(void *pointer)
```

Comment:
This function has already been defined for GT blocks. Find a common header file and move it there to avoid the duplicate code.